src/System.Runtime.Serialization.Json/tests/ReflectionOnly/Resources/System.Runtime.Serialization.Json.ReflectionOnly.Tests.rd.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="*System.Private.DataContractSerialization*">
+    <Assembly Name="System.Private.DataContractSerialization" Dynamic="Required All">
```

Comment:
@tijoytom can you please point me to the test that requires this entry? 

We shouldn't add this entry in the test project. The test project should demonstrate the actual user scenarios, which would not likely to have this entry.